tests/OmniSharp.Roslyn.CSharp.Tests/FormattingFacts.cs

```diff
@@ -128,6 +128,85 @@ public void M ()
                 new LinePositionSpanTextChange { StartLine = 2, StartColumn = 30, EndLine = 3, EndColumn = 0, NewText = "\n " });
         }
 
+
+
+
+        [Fact]
+        public async Task TextChangesOnStaringSpanBeforeFirstCharacterInLine()
+        {
+            var source = new[]
+            {
+                "class Program",
+                "{",
+                "    public static void Main()",
+                "    {",
+                "[|               int foo = 1;|]",
+                "    }",
+                "}",
+            };
+
+            await AssertTextChanges(string.Join("\r\n", source),
+                new LinePositionSpanTextChange { StartLine = 3, StartColumn = 5, EndLine = 4, EndColumn = 7, NewText = "\n" });
```

Comment:
This is really hard to read. How hard would it be to make it so the text change got applied and you could just specify the actual expected text?